runtime.md

```diff
@@ -67,28 +65,45 @@ This operation MUST return the state of a container as specified in the [State](
 In particular, the state MUST be serialized as JSON.
 
 
-### Start
+### Create
 
-`start <container-id> <path-to-bundle>`
+`create <container-id> <path-to-bundle>`
 
 This operation MUST generate an error if it is not provided a path to the bundle and the container ID to associate with the container.
-If the ID provided is not unique across all containers within the scope of the runtime, or is not valid in any other way, the implementation MUST generate an error.
-Using the data in `config.json`, that are in the bundle's directory, this operation MUST create a new container.
-This includes creating the relevant namespaces, resource limits, etc and configuring the appropriate capabilities for the container.
-A new process within the scope of the container MUST be created as specified by the `config.json` file otherwise an error MUST be generated.
+If the ID provided is not unique across all containers within the scope of the runtime, or is not valid in any other way, the implementation MUST generate an error and a new container MUST not be created.
+Using the data in [`config.json`](config.md), that is in the root of the bundle's directory, this operation MUST create a new container.
+This includes creating, or entering, the namespaces specified in the [`config.json`](config.md), resource limits, etc and configuring the appropriate capabilities for the container.
+If the `config.json` specifies that a PID namespace is to be created then one MUST be created, but the user-specified code within that namespace MUST NOT be created at this time.
+In some implementations this means that a temporary process is created in the PID namespace but it pauses until the `start` operation is invoked before replacing the process with the user-specified code.
+
+### Start
+`start <container-id>`
 
-Attempting to start an already running container MUST have no effect on the container and MUST generate an error.
+This operation MUST generate an error if it is not provided the container ID.
+This operation MUST create, and start, the user-specified code as specified by the [`process`](config.md#process-configuration) file otherwise an error MUST be generated and no process MUST be started.
```

Comment:
And I'm not sure what the “otherwise” case is for.  Is that “for any error”?  Or “when a process is not defined in `config.md`”?  With a socket-based approach, the `start` process is:

1. Reading **`process`** from `config.json`.
2. Looking up the state for the requested container ID.
3. Getting a socket path from that state.
4. Connecting to the socket.
6. Passing the process JSON through to the container process.
7. Possibly passing its standard streams through the socket to the container process, or receiving a pseudo-terminal master from the container process, or [something like that][1].
8. Exiting, or waiting on the container process and exiting with the container process' exit code (probably also [depending on how we handle standard streams][1]).

At some point, the container process will have started to apply **`process`** settings (e.g. changing it's working directory for **`process.cwd`**), and if there are errors after that point, I'm not sure it's worth unwinding them.  I'm fine explicitly listing errors that MUST happen *before* that point (e.g. if **`process.args`** isn't an array of strings, `start` should be erroring out before the container-process is processing **`process.cwd`**), but I'm not comfortable with the blanket “otherwise” you have now.


[1]: https://github.com/opencontainers/runtime-spec/pull/384/files#r59759250